ansible/roles/software/ckan/catalog/ckan-app/templates/supervisord_gunicorn.conf.j2

```diff
@@ -1,5 +1,5 @@
 [program:catalog-web]
-command=/etc/ckan/server_start.sh -b localhost:5000 --proxy-allow-from="127.0.0.1"
+command=/etc/ckan/server_start.sh -b localhost:5000 --workers=$(( 2 * `cat /proc/cpuinfo | grep 'core id' | wc -l` + 1 )) --proxy-allow-from="127.0.0.1"
```

Comment:
Let's use an Ansible variable here so that we can tune it. 2 x CPU cores is only the recommended number to start at.

Reply:
Fixed [here](https://github.com/GSA/datagov-deploy/pull/2451/commits/13988b9753b4cec3084a965ae558f37abd9a40e8) @adborden